embeddings/evaluator/submission.py

```diff
@@ -28,6 +28,58 @@ class Submission:
     packages: List[str]
     config: Optional[Dict[str, Any]] = None  # any additional config
 
+    @staticmethod
+    def _init_evaluator_kwargs(hparams: Dict[str, Any]) -> Dict[str, Any]:
+        evaluator_kwargs = {}
+        evaluation_mode = hparams.get("evaluation_mode", None)
+        tagging_scheme = hparams.get("tagging_scheme", None)
+
+        if evaluation_mode or tagging_scheme:
+            evaluator_kwargs.update(
+                {
+                    "evaluation_mode": evaluation_mode,
+                    "tagging_scheme": tagging_scheme,
+                }
+            )
+        return evaluator_kwargs
+
+    @staticmethod
+    def from_local_disk(
+        submission_name: str,
+        evaluation_file_path: T_path,
+        packages_file_path: T_path,
+        wandb_log_dir: T_path,
+        best_params_path: T_path,
+        task: str,
+    ) -> "Submission":
+        [wandb_run_dir] = [it for it in Path(wandb_log_dir).iterdir() if "run" in str(it)]
```

Comment:
```suggestion
        [wandb_run_dir] = list(Path(wandb_log_dir).glob(*run*))
```

I can suggest utilizing the `glob` method of `Path` object